BlocksForAll/ColorModViewController.swift

```diff
@@ -15,29 +15,160 @@ class ColorModViewController: UIViewController{
     var modifierBlockIndexSender: Int?
     var colorSelected: String = "white"
     
```

Comment:
lines 18 to 39:
@IBOutlet weak var blackButton: UIButton!
     @IBOutlet weak var redButton: UIButton!
     @IBOutlet weak var orangeButton: UIButton!
     @IBOutlet weak var yellowButton: UIButton!
     @IBOutlet weak var greenButton: UIButton!
     @IBOutlet weak var blueButton: UIButton!
     @IBOutlet weak var purpleButton: UIButton!
     @IBOutlet weak var whiteButton: UIButton!

 
 
 
      func deselectAll(){
         blackButton.layer.borderWidth = 0
         redButton.layer.borderWidth = 0
         orangeButton.layer.borderWidth = 0
         yellowButton.layer.borderWidth = 0
         greenButton.layer.borderWidth = 0
         blueButton.layer.borderWidth = 0
         purpleButton.layer.borderWidth = 0
         whiteButton.layer.borderWidth = 0
     }